experimental/devices/as7262/as7262.go

```diff
@@ -37,25 +38,29 @@ func New(bus i2c.Bus, opts *Opts) (*Dev, error) {
 		c:         &i2c.Dev{Bus: bus, Addr: 0x49},
 		gain:      opts.Gain,
 		interrupt: opts.InterruptPin,
-		done:      make(chan struct{}, 1),
-		timeout:   200 * time.Millisecond,
+		cancel:    func() {},
+		done:      make(chan struct{}),
```

Comment:
you should create the channel, and close it immediately.

```
d := make(chan struct{})
close(d)
...

done: d,
```

Reply:
Thats solves the timeout problem too. : )